plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/AccumuloQueryRunner.java

```diff
@@ -128,7 +128,7 @@ private static void copyTable(
                 break;
         }
 
-        LOG.info("Running import for %s", target, sql);
+        LOG.info("Running import for %s", target);
         LOG.info("%s", sql);
```

Comment:
Let's merge logs

Reply:
This is done in the next commit. I could do a more comprehensive search for logs that can be merged.